lib/acts_as_paranoid/core.rb

```diff
@@ -112,10 +118,14 @@ def destroy
           end
         end
       else
-        destroy!
+        destroy_fully!
       end
     end
 
+    def destroy
+      destroy!
```

Comment:
@zzak @mvz Sorry for the stupid question, as it's been long time since I dug inside AR internals.

Correct me if I'm wrong. That `#destroy` method uses internally `#destroy!`, which isn't the one from `ActiveRecord::Base`, but the one implemented a few lines above in the source file.

That `AAP#destroy!` updates the `deleted_at` column with the current timestamp. It doesn't remove the record from the database (like `AR::Base` does).

This last behavior is implemented by `AAP#destroy_fully!`.

To recap: by calling `AAP#destroy` we get a soft delete, which seems to be legit.

@zzak What do you think?